crates/driver/src/infra/api/routes/mod.rs

```diff
@@ -15,3 +15,34 @@ pub(super) use {
     settle::settle,
     solve::{solve, AuctionError},
 };
+
+pub fn deserialize_solution_id<'de, D>(deserializer: D) -> Result<u64, D::Error>
```

Comment:
```suggestion
pub(crate) fn deserialize_solution_id<'de, D>(deserializer: D) -> Result<u64, D::Error>
```
So the compiler helps us to remember cleaning up this function when we stop using it.